pkg/osutil/user.go

```diff
@@ -94,88 +92,102 @@ const (
 	fallbackGid  = 1000
 )
 
-var cache struct {
-	sync.Once
-	u        *user.User
-	err      error
+type Cache struct {
```

Comment:
`var cache struct {` was good, because no need to export `Cache`. I propose to revert. Maybe we can use `init()` functions to initialize this struct.

Reply:
I'm just removing the struct altogether; it doesn't really provide anything but some namespacing that we don't really need either.

```go
var (
	once     = new(sync.Once)
	limaUser *user.User
	warnings []string
)
```